climada/engine/unsequa/test/test_unsequa.py

```diff
@@ -471,6 +473,33 @@ def test_make_sample_pass(self):
             np.array(['x_exp', 'x_haz'])
             )
 
+    def test_make_sample_ff_fail(self):
+            """Test for warning and error messages when sampling using the 'ff' method"""
+
+            exp_unc, impf_unc, haz_unc = make_input_vars()
+            haz = haz_dem()
+
+            # Warning ff sampling
+            unc_calc = CalcImpact(exp_unc, impf_unc, haz_unc)
+            warning_msg = "You are using the 'ff' sampler which does not require "
+            "a value for N. The entered N value will be ignored"
+            "in the sampling process."
+
+            with self.assertLogs(ILOG, level='WARNING') as logs:
+                unc_data = unc_calc.make_sample(N=4, sampling_method='ff')
+            self.assertEqual(len(logs.output), 1)
+            self.assertIn(warning_msg, logs.output[0])
```

Comment:
I think this should be in the indention of the with. (so add one tab)
```suggestion
            self.assertEqual(len(logs.output), 1)
            self.assertIn(warning_msg, logs.output[0])
```